pkg/controller/machinehealthcheck/machinehealthcheck_controller_test.go

```diff
@@ -2515,9 +2543,9 @@ func TestHealthCheckTargets(t *testing.T) {
 		recorder := record.NewFakeRecorder(2)
 		r := newFakeReconcilerWithCustomRecorder(recorder)
 		t.Run(tc.testCase, func(t *testing.T) {
-			currentHealhty, needRemediationTargets, nextCheckTimes, errList := r.healthCheckTargets(tc.targets, tc.timeoutForMachineToHaveNode)
-			if currentHealhty != tc.currentHealthy {
-				t.Errorf("Case: %v. Got: %v, expected: %v", tc.testCase, currentHealhty, tc.currentHealthy)
+			currentHealthy, needRemediationTargets, nextCheckTimes, errList := r.healthCheckTargets(tc.targets, tc.timeoutForMachineToHaveNode, 0)
```

Comment:
Same here, by setting this to 0 we are skipping over the current logic

Reply:
Same idea here.
This is `TestHealthCheckTargets` method where our tests are in `TestReconcile` method.